ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -1027,46 +1027,43 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe
     workflow: rosa-aws-sts-hcp
-- as: aws-rosa-sts-shared-vpc-stage-critical-f7
+- as: aws-rosa-sts-shared-vpc-stage-f7
   cron: 45 4 3,10,17,24 * *
   steps:
     cluster_profile: aws-sd-qe
     env:
       CHANNEL_GROUP: nightly
-      E2E_RUN_TAGS: '@rosa and @critical'
+      E2E_RUN_TAGS: '@rosa'
       NAME_PREFIX: ocpe2e
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;ROSA&
-      TEST_IMPORTANCE: Critical
```

Comment:
It is strange when we pick up the `Critical` cases. In my memory, we picked up everything.

Reply:
I referred to one PR which is for stable.So fix it now.